examples/message_fields.q

```diff
@@ -0,0 +1,114 @@
+// message_fields.q
+// Examples of serializing from and deserializing to kdb+ data using field name dictionaries for proto messages
+
+-1"\n+----------|| message_fields.q ||----------+\n";
+
+// import the Protobuf library
+\l ../q/protobufkdb.q
+
+// Move into the protobufkdb namespace
+\d .protobufkdb
```

Comment:
Let's remove this line as functions are called with its namespace.

Reply:
done